test/actions.test.ts

```diff
@@ -182,47 +182,44 @@ describe("Test Custom SDK Logic", () => {
     });
   });
 
-  describe("(get|set)buyNowPrice", () => {
-    it("runs as expected", async () => {
-      // Set to a new value every time it's run, loop is same as current price
-      let listing: BuyNowListing = await zAuctionSdkInstance.getBuyNowListing(
-        wilderPancakesDomain
-      );
-      let newBuyNowPrice = ethers.utils.parseEther(
-        Math.round(Math.random() * 100).toString()
-      );
-      while (listing.price.eq(newBuyNowPrice)) {
-        newBuyNowPrice = ethers.utils.parseEther(
-          Math.round(Math.random() * 100).toString()
-        );
-      }
-      const params: zAuction.BuyNowParams = {
-        amount: ethers.utils.parseEther(`${newBuyNowPrice}`).toString(),
-        tokenId: wilderPancakesDomain,
-        paymentToken: wildToken,
-      };
-
-      const address = await signer.getAddress();
-      const isApproved =
-        await zAuctionSdkInstance.isZAuctionApprovedToTransferNftByDomain(
-          address,
-          wilderPancakesDomain
-        );
-
-      if (!isApproved)
-        await zAuctionSdkInstance.approveZAuctionTransferNftByDomain(
-          wilderPancakesDomain,
-          signer
-        );
+  //describe("(get|set)buyNowPrice", () => {
```

Comment:
This test no longer works.

Shall I remove it or can anyone fix it?

Reply:
Why do these tests no longer work @Will-Hyde ?

Reply:
I looked into it and it was because the most recent listing zAuction was keeping track of was from a different owner, and the new owner had not yet set a buy now. I went into the market and set one for the new owner and they work again.

You can uncomment them, and optionally add something like a `try/catch` around it to expect that message in the case owner changes again

Reply:
Added again!